promql/parser/generated_parser.y

```diff
@@ -22,7 +22,8 @@ import (
         "github.com/prometheus/prometheus/model/labels"
         "github.com/prometheus/prometheus/model/value"
         "github.com/prometheus/prometheus/model/histogram"
-	"github.com/prometheus/prometheus/promql/parser/posrange"
+
```

Comment:
This blank line causes a number of extra diffs in the generated code; is it helpful?